### PR TITLE
Fix use mTLS settings from config.yaml when flag not defined

### PR DESCRIPTION
**Pull request by Taction**
# Description

Add `sentryAddress` for mtls config. Use enableMTLS and sentryAddress from config.yaml when not supplying  from flags.

## Issue reference

Please reference the issue this PR will close: dapr/dapr#4012

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: pkg/runtime/cli.go [hunk 1]
```diff
@@ -179,8 +180,60 @@ func FromFlags() (*DaprRuntime, error) {
...
 	}
+
+	var (
+		certChain    *credentials.CertChain
+		globalConfig *global_config.Configuration
+		configErr    error
+		namespace    string
+	)
+
+	if *config != "" {
```
Comment (daixiang0) [inline pkg/runtime/cli.go:191]: Please add test cases for those if possible.

**Comment by pravinpushkar** [comment]
@Taction Could you please have a look at failing PR check so that we can proceed further. Thanks

**Comment by daixiang0** [comment]
@Taction please fix conflicts then we can make it land.

**Comment by cecilphillip** [comment]
I ran into this same issue this week when trying to setup sentry in docker-compose

**Comment by Taction** [comment]
Since there are several conflicts, which is a bit complicated to solve, moved the code to #6347

**Pull request closed without merging**